lib/main.js

```diff
@@ -26,6 +26,11 @@ function loadDeps() {
 // Local variables
 const parseRegex = /(\d+):(\d+):\s(([A-Z])\d{2,3})\s+(.*)/g;
 const execPathVersions = new Map();
+const defaultConfigFiles = [
```

Comment:
Is there a reason this is statically declared here, instead of setting this as the default value for the `configFile` option? Should we care if the config file found is a default one Flake8 handles, or one specified by the user?

Reply:
If we set these files as the default value of `configFile` then we also need to maintain the order in which `flake8` is supposed to be looking for them. This seemed to be something that changed already in the past, as you mentioned in https://github.com/AtomLinter/linter-flake8/issues/554#issuecomment-353121371.

Instead, calling `flake8` without the `--config` parameter makes it choose the default config file in the correct order. I believe this is a more robust approach than us hardcoding the order those default config files need to be chosen.